server/spec/entitlement_spec.rb

```diff
@@ -83,6 +83,14 @@
     @system.list_entitlements.first.quantity.should == 10
   end
 
+  it 'should allow consumption of quantity zero' do
+    pool = find_pool_for_product @virt
+    pool['consumed'].should eq(0)
+    ent = @system.consume_pool(pool.id, {:quantity => 0})
```

Comment:
@Ceiu bindng by zero is allowed today. I think it is meant to represent bind by 1.
I dont know why this is allowed, or if it should be blocked. I thought it is safe to keep the functionality since we explicitly allow it in [this](https://github.com/candlepin/candlepin/blob/master/server/src/main/java/org/candlepin/model/Entitlement.java#L152) code that has always been there.